src/storage/mvcc/txn.rs

```diff
@@ -594,6 +600,73 @@ impl<S: Snapshot> MvccTxn<S> {
         })
     }
 
+    /// Check the status of a transaction.
+    ///
+    /// This operation checks whether a transaction has expired it's Lock's TTL, rollback the
+    /// transaction if expired, and update the transaction's min_commit_ts according to the metadata
+    /// in the primary lock.
+    ///
+    /// When transaction T1 meets T2's lock, it may invoke this on T2's primary key. In this
+    /// situation, `self.start_ts` is T2's `start_ts`, `caller_start_ts` is T1's `start_ts`, and
```

Comment:
So here t2 is the transaction's status we want to check?

Reply:
Yes.